catch/include/cmd_options.hh

```diff
@@ -23,11 +23,11 @@ THE SOFTWARE.
 #pragma once
 
 struct CmdOptions {
-  int iterations = 1000;
+  int iterations = 10;
```

Comment:
This change requires some explanation and may need proof of a consensus from the HIP leads.

Reply:
@mangupta @gargrahul please review this change to the number of iterations.